.github/workflows/pull-request.yml

```
@@ -279,20 +279,44 @@
+          zip -r app.ipa Payload
+        # This packages the Runner.app into an .ipa file
+
+      - uses: ncipollo/release-action@v1
+        with:
+          name: "Automated iOS Release"
+          artifacts: "app-release.ipa"
+          allowUpdates: "true"
+          generateReleaseNotes: false
+          tag: "automated"
+          body: |
+            This is an automated release, triggered by a recent push. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)